### ISSUE TITLE
Angular 16 support

### ISSUE THREAD

**Issue by plessbd** [amitdahan/ngx-filesize#423]
As the title says, angular 16 support

**Comment by amitdahan** [comment]
Hey! Thanks for the issue @plessbd !

Is there any specific error message when you tried using `ngx-filesize` with Angular 16?
I just did a small attempt and it just worked with no apparent issues:
https://stackblitz.com/edit/angular-sl1btb

Anything I missed?

**Comment by plessbd** [comment]
I have only had issues installing:

```
% npm i
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: frontend@3.1.0
npm ERR! Found: @angular/common@16.0.3
npm ERR! node_modules/@angular/common
npm ERR!   @angular/common@"^16.0.2" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @angular/common@">= 14.2.0 < 16.0.0" from ngx-filesize@3.0.1
npm ERR! node_modules/ngx-filesize
npm ERR!   ngx-filesize@"^3.0.1" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR!
npm ERR! For a full report see:
npm ERR! /Users/be38643/.npm/_logs/2023-05-28T15_57_26_038Z-eresolve-report.txt
```
however using `--legacy-peer-deps` I have not had any issues.

I think it is just an update to the  package.json peer dep, I did a PR for that, never build a module like this so not sure if that is enough

**Comment by mhaertwig** [comment]
@amitdahan, the error is due to the configured `peerDependencies` in the `package.json` that need to be updated: 
https://github.com/amitdahan/ngx-filesize/blob/c7b0c2ba1c115acd19dcad4bf52ec2f36e673090/packages/ngx-filesize/package.json#L40C4-L44

Let me know if you need a PR for that.

**Comment by amitdahan** [comment]
Released `3.0.2` which should include @plessbd's fix 🎉
